SendLog.py

```diff
@@ -93,15 +111,74 @@ def get_line_and_state(file):
 		# Add figure ID to list
 		figure_list.append(figure_id)
 		return (None, INITIALIZATION)
+        elif state == BLOCK_IDENTIFICATION:#check for block identification 
+                return (line,BLOCK_IDENTIFICATION) 
 	elif state == ENDING:
```

Comment:
There is a difference in the indentation between this line and ```line 114```, please fix and maintain consistency.